udev/rules.d/.gitignore

```diff
@@ -1,4 +1 @@
-69-vdev.rules
-60-zpool.rules
-60-zvol.rules
-90-zfs.rules
+*.rules
```

Comment:
nit: a good but unrelated fix which seems to have ended up in these changes.  Otherwise, looks good to me.

Reply:
removed